packages/@ember/-internals/glimmer/lib/environment.ts

```diff
@@ -99,6 +99,12 @@ export default class Environment extends GlimmerEnvironment {
 
     try {
       super.commit();
+    } catch(error) {
+      if (completedWithoutError) {
+        throw error;
+      } else {
+        console.error(error);
```

Comment:
So, if I'm reading this correctly, if `completedWithoutError` is true then `error` comes from the commit itself. What is the `error` if `completedWithoutError` is false? Is it the error of the outer `try/catch`?

Reply:
Correct. Basically we were already planning to throw a different error from where we are called.

Reply:
Ahhh I see now, ok 👍